bin/openssh-ldap-publickey

```diff
@@ -212,7 +217,7 @@ unless (
 my $log_level = $conf->{'openssh_ldap_loglevel'};
 my $logfile = $conf->{'openssh_ldap_logfile'} || $fallback_logfile;
 my $timeout = $conf->{'timeout'} || 10;
-my $nss_base_passwd = $conf->{'nss_base_passwd'} || [ { nss_base_passwd => "ou=People," .$conf->{'base'}, scope => undef } ];
+my $nss_base_passwd = $conf->{'nss_base_passwd'} || [ { nss_base_passwd => $conf->{'base'}, scope => undef } ];
```

Comment:
i think it might brake backward compatibility for an existing clients. It's fine to drop the `ou=People,` part but we need to introduce some transitioning config variable for that. Something like: 
```
my $nss_base = $conf->{'nss_base'} || $conf->{'base'}
my $nss_base_passwd = $conf->{'nss_base_passwd'} || [ { nss_base_passwd => $nss_base, scope => undef } ];
```

without setting default value for `$conf->{'nss_base'}`